docs/api/instruments/agilent/agilent8648.rst

```diff
@@ -0,0 +1,7 @@
+##############################################
+Agilent 8648 Signal Generator
+##############################################
```

Comment:
Please make the hashes as long as the text.

Reply:
Fixed